packages/functional-tests/pages/login.ts

```diff
@@ -132,6 +132,14 @@ export class LoginPage extends BaseLayout {
     return this.page.locator(selectors.SUBMIT);
   }
 
+  get signUpCodeHeader() {
+    return this.page.locator(selectors.SIGN_UP_CODE_HEADER);
+  }
+
+  get signInCodeHeader() {
```

Comment:
And this one to signinTokenCode?

Reply:
I suggest to switch over to these POMS in an independent task